packages/lib/services/interop/InteropService_Exporter_Md.ts

```diff
@@ -124,15 +131,36 @@ export default class InteropService_Exporter_Md extends InteropService_Exporter_
 
 			const noteBody = await this.relaceLinkedItemIdsByRelativePaths_(item);
 			const modNote = Object.assign({}, item, { body: noteBody });
-			const noteContent = await Note.serializeForEdit(modNote);
+			const noteContent = await this.getNoteExportContent_(modNote);
 			await shim.fsDriver().mkdir(dirname(noteFilePath));
 			await shim.fsDriver().writeFile(noteFilePath, noteContent, 'utf-8');
 		}
 	}
 
+	async findReasonableFilename(_resource: any, filePath: string) {
+		let fileName = basename(filePath);
+
+		if (_resource.filename) {
```

Comment:
I think it should be `resource` because `_resource` normally means the function argument is not used.